packages/expo-dev-launcher/write_embedded_bundle.sh

```diff
@@ -9,7 +9,7 @@ export EXPO_NO_TELEMETRY=1
 cat > app.json <<EOF
 {
   "expo": {
-    "jsEngine": "jsc"
+    "jsEngine": "hermes"
```

Comment:
Won't this change compatibility with apps that use JSC?

Reply:
Yes, it's a temporary change. We can revert now after #30763 landed